src/ai/backend/manager/cli/image_impl.py

```diff
@@ -118,6 +118,27 @@ async def forget_image(cli_ctx, canonical_or_alias, architecture):
                     ImageAlias(canonical_or_alias),
                 ],
             )
+            await image_row.mark_as_deleted(session)
+        except UnknownImageReference:
+            log.exception("Image not found.")
+        except Exception:
+            log.exception("An error occurred.")
```

Comment:
It seems difficult to find what kind of error it is in the `Exception`. Is there a better way to do it?